service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -45,6 +47,13 @@ class CaseData
     #[Annotation\Validator(Explode::class, options: ['validator' => ['name' => LpaUidValidator::class]])]
     public array $lpas;
 
+    public ?string $kbvQuestions = null;
+
+    #[Annotation\Required(false)]
+    #[Annotation\Validator(IsType::class, options: ['type' => 'boolean'])]
+    #[Annotation\Validator(NotEmpty::class, options: [NotEmpty::NULL])]
```

Comment:
Maybe i'm missing something, this looks like it would have the same affect as line 52?

Reply:
It's a quirk of laminas-form that I don't fully understand: even if you say the field isn't required, it still seems to apply the `NotEmpty` filter and fails in surprising ways.